apps/etterlatte-institusjonsopphold/src/main/kotlin/kafka/KafkaConsumerInstitusjonsopphold.kt

```diff
@@ -19,7 +20,13 @@ class KafkaConsumerInstitusjonsopphold(
         pollTimeoutInSeconds = Duration.ofSeconds(10L),
     ) {
     override fun stream() {
-        stream { meldinger -> meldinger.forEach { behandlingKlient.haandterHendelse(it) } }
+        stream { meldinger ->
+            meldinger.forEach {
+                runBlocking {
```

Comment:
Den kjører jo en og en i dag uten runBlocking rundt en suspend funksjon, skjønner ikke helt denne ?

Reply:
For ordens skyld svarer jeg her også i tillegg til Slack. 

Jeg gjør et forsøk på å gjøre oppsettet likt de andre hendelsesappene. Dette siden jeg vet at de andre ikke hopper over hendelser dersom noe går galt.